pkg/lightning/backend/tidb/tidb.go

```diff
@@ -333,13 +363,21 @@ func NewTiDBBackend(
 		log.FromContext(ctx).Warn("unsupported conflict strategy for TiDB backend, overwrite with `error`")
 		onDuplicate = config.ErrorOnDup
 	}
+	stmtCache := kvcache.NewSimpleLRUCache(prepStmtCacheSize, 0, 0)
```

Comment:
I don't know what's the meaning of the parameters `guard` and `quota` 😂 I'll learn them later.

And no need to create the LRU cache when `LogicalImportPrepStmt` is false? Or we can use nilness of `stmtCache` as the bool `cachePrepStmts`?

Reply:
Please address this comment. Currently caching prep statement and using prep statement are the same thing, we don't need two members.